app/addons/documents/changes/components.react.jsx

```diff
@@ -134,15 +135,30 @@ define([
       return changesStore.hasFilter(filter);
     },
 
+    togglePolling: function () {
+      Actions.togglePolling();
+    },
+
     render: function () {
       return (
         <div className="tab-content">
           <div className="tab-pane active" ref="filterTab">
             <div className="changes-header js-filter">
+              <div className="changes-polling">
+                <input
+                  type="checkbox"
+                  id="changes-toggle-polling"
+                  checked={this.state.pollingEnabled ? 'checked="checked"': null}
```

Comment:
i think this can be:  `checked={this.state.pollingEnabled}`  

https://facebook.github.io/jest/docs/tutorial-react.html